node/jaeger/src/spans.rs

```diff
@@ -277,7 +277,7 @@ impl Span {
 	}
 
 	/// Derive a child span from `self`.
-	pub fn child(&self, name: &'static str) -> Self {
+	pub fn child(&self, name: &str) -> Self {
```

Comment:
This could still be static if you use match / if let when creating the span.

Reply:
Yes, it can be static, but it doesn't makes sense to be starting here since inner.child doesn't have this restriction.